### ISSUE TITLE
Feature Request: add specific directories to full-backup

### ISSUE THREAD

**Issue by XxJooO** [simatec/ioBroker.backitup#154]
I'm using a debian 9.9 machine and it would be nice to choose a list of directories to be added to a full backup.

e.g. I want to backup my influxDB data and my grafana installation so it would be nice to add the directories /var/lib/influxdb and var/lib/grafana

**Comment by simatec** [comment]
You can make a specific path in the NAS settings.  choose the option "Copy" in the NAS settings

**Comment by XxJooO** [comment]
Sorry simatec, I think you did not understand what I want to do. I know that I can specify the directories where I want to copy "TO" on the NAS. 
But I want to choose directories from the source (that means from my ioBroker Linux installation) "FROM" where I want to copy to the NAS.

**Comment by simatec** [comment]
Hello, this will not support backitup.  The local home directory for backups is the foundation of Backitup.  In one of the next versions, there will still be support for backing up benches with path information.  maybe this will help you

**Comment by XxJooO** [comment]
simatec, thank you for your answer, this would be great. But I could tell you that I found an "interim"-solution: I created a subdirectory "/opt/iobroker/influxdb" and via the config-file of influxdb I could tell the program that all config-files of influxDB and all data of the databases are stored there and backitup makes a full backup even of these directories and the content of them.
So its great at all!
